scripts/demo.with_docker.sh

```diff
@@ -42,6 +42,6 @@ sudo docker run --runtime=nvidia --rm \
   -e "ELMO_SRC_DIR=$ELMO_SRC_DIR" \
   -e "WORD_EMBS_FILE=$WORD_EMBS_FILE" \
   -e "JIANT_PROJECT_PREFIX=$NFS_PATH/exp/$USER" \
-  -e "PYTORCH_PRETRAINED_BERT_CACHE=$PYTORCH_PRETRAINED_BERT_CACHE" \
+  -e "PYTORCH_PRETRAINED_BERT_CACHE=$PYTORCH_TRANSFORMERS_CACHE" \
```

Comment:
Commented on this earlier - looks like it still uses the old name: https://github.com/huggingface/pytorch-transformers/blob/4bcddf6fc800a11201608b9821e4c0b55a27eb57/pytorch_transformers/file_utils.py#L40

Can we keep it consistent in our code?

Reply:
Oops—failed replacement. The intent is to consistently use the environment variable name in scripts, but the more informative lowercased internal name in Python code.